index.html

```diff
@@ -404,7 +404,7 @@
               <a for="RemotePlabyackState">connecting</a>.
             </li>
             <li>
-              <a>Resolve</a> <var>promise</var> with <code>true</code>.
+              Fulfilled <var>promise</var> with <code>true</code>.
```

Comment:
nit: I think in the algorithms we use infinitive form of the verb, so it should be Fulfill rather than Fulfilled here and in a few places below.